app/controllers/api/hud_filters_controller.rb

```diff
@@ -30,11 +30,13 @@ def index
           :computed_project_type,
           o_t[:OrganizationName],
           o_t[:id],
-        ).each do |id, p_name, p_confidential, type, o_name, o_id|
-          name = GrdaWarehouse::Hud::Project.confidentialize_name(current_user, p_name, p_confidential)
-          @data[[o_id, o_name]] ||= []
-          @data[[o_id, o_name]] << [
-            "#{name} (#{HUD.project_type_brief(type)})",
+          o_t[:confidential],
+        ).each do |id, p_name, p_confidential, type, o_name, o_id, o_confidential|
+          project_name = GrdaWarehouse::Hud::Project.confidentialize_name(current_user, p_name, p_confidential || o_confidential)
+          organization_name = o_confidential ? GrdaWarehouse::Hud::Organization.confidential_organization_name : o_name
+          @data[[o_id, organization_name]] ||= []
+          @data[[o_id, organization_name]] << [
+            "#{project_name} (#{HUD.project_type_brief(type)})",
```

Comment:
This is updated to also confidentialize the Organization name if applicable, which we weren't doing before. 
<img width="831" alt="Screen Shot 2022-06-10 at 2 25 13 PM" src="https://user-images.githubusercontent.com/5333982/173128206-d1734545-5a68-47d0-b7ee-763424621e0e.png">